minio/base/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: minio
+commonLabels:
+  app: minio
+  component: minio
```

Comment:
You can drop this `component` label since you're not managing dex from the same kustomization file like I was in the demo repository.

Reply:
Sounds good, I made this change.